src/reader/TranslatableWord.js

```diff
@@ -35,7 +35,11 @@ export default function TranslatableWord({
   }, [word]);
 
   function clickOnWord(e, word) {
-    if (word.token.is_like_num || word.token.is_punct) return;
+    if (
+      word.token.is_like_num ||
```

Comment:
I see how it might make sense to have a token object... we could define helper functions for this kind of situations.